src/Elliptic/FirstOrderOperator.hpp

```diff
@@ -0,0 +1,71 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "NumericalAlgorithms/LinearOperators/Divergence.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace elliptic {
+
+/*!
+ * \ingroup ActionsGroup
+ * \ingroup DiscontinuousGalerkinGroup
+ * \brief Compute the bulk contribution to the operator represented by the
+ * `StepPrefixTag` applied to the `VarsTag`.
```

Comment:
What is a StepPrefixTag? What possible values can it take? Please specify here or point to relevant docs

Reply:
Added more details to the docs

Reply:
I still don't understand what a StepPrefixTag is.  Can you point to an equation illustrating it?

Reply:
pushed a fixup to address this, please take a look